easypy/concurrency.py

```diff
@@ -1,9 +1,65 @@
 # encoding: utf-8
 
 """
-This module helps you run things concurrently
+This module helps you run things concurrently.
+Most useful are the ``concurrent`` context manager and the ``MultiObject`` class.
+The features in this module are integrated with the ``logging`` module, to provide
+thread-context to log messages. It also has support for integration ``gevent``.
+
+``MultiObject``
+
+Here's how you would, for example, concurrently send a message to a bunch of servers::
+
+    responses = MultiObject(servers).send('Hello')
```

Comment:
not accurate. here `responses` is of `MultiObject` type...

Reply:
I think that's too pedantic for this area of this documentation. better to keep it simple and broad.